presets/4.5/filters/rubberquads/squirrel_35_filter.txt

```diff
@@ -0,0 +1,93 @@
+#$ TITLE: 4.5 Filter settings for the RubberQuads Squirrel 35 (with RPM FILTERING).
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: FILTERS
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: rubberquads, squirrel, DShot telemetry, filter, filters, clean, basic, betaflight
+#$ AUTHOR: Volker Göschl-Covrljan
+#$ DISCUSSION: https://discord.gg/RqbQBuh7pu
+
+#$ PARSER: MARKED
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: <img src="https://www.rubberquads.com/static/rubberquads/RQ-logo_header.png" width="200px" style="margin-left: auto; margin-right: auto; display: block;"/>
+#$ DESCRIPTION:
+#$ DESCRIPTION: # Universal Filter Settings for 3"-4" rigs
+#$ DESCRIPTION:
+#$ DESCRIPTION: WARNING: Requires RPM filtering.  Ensure that bidirectional DShot working!
+#$ DESCRIPTION: WARNING: May cause motor overheating -> Check temps after 30 seconds!
+#$ DESCRIPTION: Intended for solid frames and clean props - will growl but fly home with bended props.
+#$ DESCRIPTION: If motors get too hot, lower both filter-sliders by 0.2 and try again.
+
+
+
+#$ INCLUDE: presets/4.5/filters/defaults.txt
+
+# enable dshot rpm telemetry
+set dshot_bidir = ON
+
+# -- Gyro filters --
+# Standard dynamic gyro lowpass cascade
```

Comment:
i think most of the values "gyro_lpf..." not needed, since they are default, or covered by gyro filter slider?

Reply:
5 is default but with smaller stators a bit more expo is needed due to slower ramp-up so I leave just the expo (in case I need to change it later)